docs/gameroom_walkthrough/act_1/scene_1.md

```diff
@@ -0,0 +1,26 @@
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+## Scene 1: Alice logs into Acme's Gameroom application
+
+Alice looks at the GAMEROOM APP LOGIN SCREEN in her browser.
```

Comment:
Looks like you accidentally added this file back (it has been moved to each release and combined with the background part)

Reply:
Updated